qualification/antenna_channelised_voltage/test_delay.py

```diff
@@ -657,16 +665,25 @@ async def measure_once(freqs: tuple[float, float]) -> tuple[float, float, float]
             Standard deviation in the delay estimate. Note that the distribution is
             non-Gaussian and is due to quantisation error, which is bounded.
         """
-        signal = f"cw({amplitude}, {freqs[0]}); cw({amplitude}, {freqs[1]});"
-        await cbf.dsim_clients[0].request("signals", signal, dsim_period)
-        pdf_report.detail(f"dsim signal set to {signal}.")
+        async with asyncio.TaskGroup() as tg:
+            for dsim_client, dsim_freqs in zip(cbf.dsim_clients, cfreqs_by_dsim):
+                if dsim_freqs:
+                    signal = " ".join(
+                        f"multicw({len(dsim_freqs)}, {amplitude}, 0.0, {dsim_freqs[0] + rfreq}, {freq_step_dsim});"
+                        for rfreq in rel_freqs
```

Comment:
Going along with my above comment:

```suggestion
                        f"multicw({len(dsim_freqs)}, {amplitude}, 0.0, {dsim_freqs[0] + rfreq}, {freq_step_dsim});"
                        for rfreq in [0, frequency_delta]
```

Reply:
That would change the second pass, which currently tests a symmetric range around the centre of the channel.